.github/workflows/release.yml

```diff
@@ -128,3 +128,6 @@ jobs:
           omitNameDuringUpdate: true
           omitPrereleaseDuringUpdate: true
           skipIfReleaseExists: true
+
+      - name: Publish doc
+        uses: ./.github/actions/publish-doc
```

Comment:
This is untested, but should work? It works for the CI workflow: https://github.com/bytecodealliance/wasmtime-rb/blob/b5139ef7565e3f4e9671387db5c79f661b112f9d/.github/workflows/ci.yml#L118-L120